### PR TITLE
Tracer fixes

### PR DESCRIPTION
**Pull request by ogajduse**
This PR should address all concerns that were brought up in #748.

### CODE REVIEW

**Comment by mirekdlugosz** [comment]
Why is this draft?
Looks like it only adds new unit tests, and they did pass on travis, so we are all good here?

**Comment by ogajduse** [comment]
I am still working on some changes that need to be done in order to satisfy all comments in #748.

**Review by rplevka — APPROVED** [review]

**Comment by swadeley** [comment]
Hello

I have fetched and checked out this branch. 

nailgun]$ git status
On branch pr/758

Running test that failed here:
https://github.com/SatelliteQE/nailgun/pull/748#issuecomment-706567689

robottelo]$ pytest --show-capture=no tests/foreman/api/test_errata.py::ErrataTestCase::test_positive_get_count_for_host
=========================== test session starts ================================
platform linux -- Python 3.7.9, pytest-5.4.3, py-1.9.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: sat6/robottelo_2/robottelo
plugins: ibutsu-1.0.34, services-1.3.1, mock-1.10.4, forked-1.3.0, xdist-1.34.0
collecting ... 2020-10-13 14:56:48 - conftest - DEBUG - Collected 1 test cases
collected 1 item                                                                                                 

tests/foreman/api/test_errata.py .                                                                         [100%]

==================== 1 passed, 2 warnings in 897.75s (0:14:57) ====================

File: nailgun/entities.py [hunk 1]
```diff
@@ -4038,7 +4039,7 @@ def errata(self, synchronous=True, **kwargs):
         return _handle_response(response, self._server_config, synchronous)
 
     def traces(self, synchronous=True, **kwargs):
-        """List services that need restarting on the host
+        """List services that need restarting for the host
```
Comment (swadeley) [inline nailgun/entities.py:4042]: In this case, _on_ the host was OK (because that is where the services run). But its not important.

**Review by swadeley — APPROVED** [review]
ACK

**Review by mirekdlugosz — APPROVED** [review]
ACK

**Pull request merged**